test/integration/src/test/java/io/pravega/test/integration/ControllerMetricsTest.java

```diff
@@ -207,6 +210,23 @@ public void streamMetricsTest() {
         checkStatsRegisteredValues(iterations * iterations, UPDATE_STREAM_LATENCY, TRUNCATE_STREAM_LATENCY);
     }
 
+    /**
+     * This test verifies that the Controller increments the metric for Zookeeper session expiration events correctly.
+     *
+     * @throws Exception
+     */
+    @Test(timeout = 60000)
+    public void zookeeperMetricsTest() throws Exception {
+        Counter zkSessionExpirationCounter = MetricRegistryUtils.getCounter(getCounterMetricName(globalMetricName(CONTROLLER_ZK_SESSION_EXPIRATION)));
+        Assert.assertNull(zkSessionExpirationCounter);
+        controllerWrapper.forceClientSessionExpiry();
+        while (zkSessionExpirationCounter == null) {
+            Thread.sleep(1000);
```

Comment:
Let's make this shorter to speed up the test run.

Reply:
Done.